examples/asr/speech_to_text_bpe.py

```diff
@@ -59,8 +59,13 @@ def main(cfg):
     logging.info(f'Hydra config: {cfg.pretty()}')
     trainer = pl.Trainer(**cfg.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
+
     asr_model = EncDecCTCModelBPE(cfg=cfg.model, trainer=trainer)
 
+    if cfg.model.get("load_weights_from_checkpoint", None):
+        logging.info(f"Loading checkpoint '{cfg.model.load_weights_from_checkpoint}' ...")
+        asr_model.update_weights(checkpoint_path=cfg.model.load_weights_from_checkpoint)
+
     trainer.fit(asr_model)
 
     if hasattr(cfg.model, 'test_ds') and cfg.model.test_ds.manifest_filepath is not None:
```

Comment:
If you dont mind, could you update this part same as above, or I can do it in a later PR.

Reply:
Done.